core/dbt/adapters/base/connections.py

```diff
@@ -248,7 +248,10 @@ def commit(self):
     @classmethod
     def _rollback_handle(cls, connection):
         """Perform the actual rollback operation."""
-        connection.handle.rollback()
+        try:
+            connection.handle.rollback()
+        except Exception:
+            logger.exception('Failed to rollback {}'.format(connection.name))
```

Comment:
Can we use `logger.error` instead of `logger.exception` here? We try not to log stack traces to stdout. In this case, it might be appropriate to log the exception to the debug logs, eg. with `logger.debug('', exc_info=True)`.

Reply:
> Can we use `logger.error` instead of `logger.exception` here? We try not to log stack traces to stdout. In this case, it might be appropriate to log the exception to the debug logs, eg. with `logger.debug('', exc_info=True)`.

Makes sense! I'll have an update shortly.